src/renderer/utils/utils.ts

```diff
@@ -96,6 +96,29 @@ export const useFocusOutside = <T extends HTMLElement = HTMLElement>(
   }, [handleFocus]);
 };
 
+export const useFocusOut = <T extends HTMLElement = HTMLElement>(
+  ref: RefObject<T>,
+  callback: () => void
+) => {
+  const handleFocus = useMemo(() => {
+    return (e: globalThis.FocusEvent) => {
+      if (
+        ref.current &&
+        ref.current.contains(e.target as Node) &&
+        !ref.current.contains(e.relatedTarget as Node)
+      ) {
+        callback();
+      }
+    };
+  }, [callback, ref]);
+
+  useEffect(() => {
+    document.addEventListener('focusout', handleFocus, true);
```

Comment:
Can we instead apply the event listener to `ref.current` (`ref.current.addEventListener`) and then simplify `handleFocus` to be:
```
const handleFocus = useMemo(() => {
    return (e: globalThis.FocusEvent) => {
      if (e.currentTarget.contains(e.target as Node)) {
        callback();
      }
    };
}, [callback]);
```

Reply:
Whoops nvm, I misread and thought you were referring to the `useFocusOutside` function. This change seems to work so I'll go ahead and make it.